devfiles/lombok/devfile.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: 1.0.0
+metadata:
+  generateName: javalombok-
+projects:
+  - name: lombok-demo
+    source:
+      location: 'https://github.com/che-samples/lombok-demo.git'
+      type: git
+      branch: main
+components:
+  - id: redhat/java/latest
+    type: chePlugin
+    volumes:
+      - containerPath: /tmp/vscode-unpacked
+        name: lombok-java-unpacked
+    preferences:
+      java.jdt.ls.vmargs: -javaagent:/tmp/vscode-unpacked/lombok.jar
+  - type: chePlugin
+    id: cool2k/vscode-lombok/latest
+    volumes:
+      - containerPath: /tmp/vscode-unpacked
+        name: lombok-java-unpacked
```

Comment:
@gattytto This looks tricky for the enduser to figure out what he needs to do to have the plugin working.
Why not making it in the same sidecar with Java extension like we do for Quarkus?

Reply:
the other option we could try: add the volume `vscode-unpack` to all the java based plugin.
and add 
```
 preferences:
      java.jdt.ls.vmargs: -javaagent:/tmp/vscode-unpacked/lombok.jar
```
to the lombok meta.yaml (not sure it is possible)

Reply:
@sunix that is a [feature request](https://github.com/eclipse/che/issues/17975)